modules/virt-about-vmis.adoc

```diff
@@ -23,3 +23,5 @@ When you delete a VM, the associated VMI is automatically deleted. You delete a
 ====
 Before you uninstall {VirtProductName}, list and view the standalone VMIs by using the CLI or the web console. Then, delete any outstanding VMIs.
 ====
+
+When you edit a VM, some settings may be applied to the VMIs dynamically and without the need for a restart. Any change made to a VM object that cannot be applied to the VMIs dynamically will trigger the `RestartRequired` VM condition. Changes are effective on the next reboot, and the condition is removed.
```

Comment:
```suggestion
When you edit a VM, some settings might be applied to the VMIs dynamically and without the need for a restart. Any change made to a VM object that cannot be applied to the VMIs dynamically will trigger the `RestartRequired` VM condition. Changes are effective on the next reboot, and the condition is removed.
```
Let's fix "may" because it doesn't translate correctly.